subprojects/plugins/src/integTest/groovy/org/gradle/groovy/compile/BasicGroovyCompilerIntegrationSpec.groovy

```diff
@@ -134,7 +134,7 @@ abstract class BasicGroovyCompilerIntegrationSpec extends MultiVersionIntegratio
 
         buildFile << """
             compileGroovy {
-                options.compilerArgs << '-proc:none'
+                options.annotationProcessorPath = files()
```

Comment:
❌ No change should be necessary here - `proc:none` will still disable processing, so no classes will be generated.

Reply:
With `-proc:none` it fails:

```
Using 8 worker leases.
Starting Build
Compiling initialization script '/var/folders/k0/459h8z3d6pj1ybn5q_wxx98w0000gn/T/mirrors6787554305939794493.gradle' using SubsetScriptTransformer.
Compiling initialization script '/var/folders/k0/459h8z3d6pj1ybn5q_wxx98w0000gn/T/mirrors6787554305939794493.gradle' using BuildScriptTransformer.
Compiling settings file '/Users/marc/Repositories/gradle/gradle/build/tmp/test files/InProcessGroovyCompilerIntegrationTest/jointCompileBadCode...rDisabled/f1nto/settings.gradle' using SubsetScriptTransformer.
Compiling settings file '/Users/marc/Repositories/gradle/gradle/build/tmp/test files/InProcessGroovyCompilerIntegrationTest/jointCompileBadCode...rDisabled/f1nto/settings.gradle' using BuildScriptTransformer.
Settings evaluated using settings file '/Users/marc/Repositories/gradle/gradle/build/tmp/test files/InProcessGroovyCompilerIntegrationTest/jointCompileBadCode...rDisabled/f1nto/settings.gradle'.
Projects loaded. Root project using build file '/Users/marc/Repositories/gradle/gradle/build/tmp/test files/InProcessGroovyCompilerIntegrationTest/jointCompileBadCode...rDisabled/f1nto/build.gradle'.
Included projects: [root project 'f1nto', project ':processor']

> Configure project :
Evaluating root project 'f1nto' using build file '/Users/marc/Repositories/gradle/gradle/build/tmp/test files/InProcessGroovyCompilerIntegrationTest/jointCompileBadCode...rDisabled/f1nto/build.gradle'.
Compiling build file '/Users/marc/Repositories/gradle/gradle/build/tmp/test files/InProcessGroovyCompilerIntegrationTest/jointCompileBadCode...rDisabled/f1nto/build.gradle' using SubsetScriptTransformer.
Compiling build file '/Users/marc/Repositories/gradle/gradle/build/tmp/test files/InProcessGroovyCompilerIntegrationTest/jointCompileBadCode...rDisabled/f1nto/build.gradle' using BuildScriptTransformer.

> Configure project :processor
Evaluating project ':processor' using build file '/Users/marc/Repositories/gradle/gradle/build/tmp/test files/InProcessGroovyCompilerIntegrationTest/jointCompileBadCode...rDisabled/f1nto/processor/build.gradle'.
Compiling build file '/Users/marc/Repositories/gradle/gradle/build/tmp/test files/InProcessGroovyCompilerIntegrationTest/jointCompileBadCode...rDisabled/f1nto/processor/build.gradle' using SubsetScriptTransformer.
Compiling build file '/Users/marc/Repositories/gradle/gradle/build/tmp/test files/InProcessGroovyCompilerIntegrationTest/jointCompileBadCode...rDisabled/f1nto/processor/build.gradle' using BuildScriptTransformer.
All projects evaluated.
Selected primary task 'compileGroovy' from project :
Tasks to be executed: [task ':processor:compileJava', task ':processor:processResources', task ':processor:classes', task ':processor:jar', task ':compileJava', task ':compileGroovy']
:processor:compileJava (Thread[Execution worker for ':' Thread 6,5,main]) started.

> Task :processor:compileJava
Task ':processor:compileJava' is not up-to-date because:
  No history is available.
All input files are considered out-of-date for incremental task ':processor:compileJava'.
Full recompilation is required because no incremental change information is available. This is usually caused by clean builds or changing compiler arguments.
Compiling with JDK Java compiler API.
getOrCreateProvider(BC) created instance of org.bouncycastle.jce.provider.BouncyCastleProvider
Created classpath snapshot for incremental compilation in 0.005 secs.
:processor:compileJava (Thread[Execution worker for ':' Thread 6,5,main]) completed. Took 1.671 secs.
:processor:processResources (Thread[Execution worker for ':' Thread 6,5,main]) started.

> Task :processor:processResources
Task ':processor:processResources' is not up-to-date because:
  No history is available.
:processor:processResources (Thread[Execution worker for ':' Thread 6,5,main]) completed. Took 0.03 secs.
:processor:classes (Thread[Execution worker for ':' Thread 6,5,main]) started.

> Task :processor:classes
Skipping task ':processor:classes' as it has no actions.
:processor:classes (Thread[Execution worker for ':' Thread 6,5,main]) completed. Took 0.0 secs.
:processor:jar (Thread[Execution worker for ':' Thread 6,5,main]) started.

> Task :processor:jar
Task ':processor:jar' is not up-to-date because:
  No history is available.
:processor:jar (Thread[Execution worker for ':' Thread 6,5,main]) completed. Took 0.054 secs.
:compileJava (Thread[Execution worker for ':' Thread 6,5,main]) started.

> Task :compileJava NO-SOURCE
file or directory '/Users/marc/Repositories/gradle/gradle/build/tmp/test files/InProcessGroovyCompilerIntegrationTest/jointCompileBadCode...rDisabled/f1nto/src/main/java', not found
Skipping task ':compileJava' as it has no source files and no previous output files.
:compileJava (Thread[Execution worker for ':' Thread 6,5,main]) completed. Took 0.001 secs.
:compileGroovy (Thread[Execution worker for ':' Thread 6,5,main]) started.

> Task :compileGroovy
Failed to get resource: GET. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/apache/ant/ant/1.9.13/ant-1.9.13.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/apache/ant/ant-parent/1.9.13/ant-parent-1.9.13.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/commons-cli/commons-cli/1.4/commons-cli-1.4.pom]
Failed to get resource: GET. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/info/picocli/picocli/3.6.0/picocli-3.6.0.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/com/thoughtworks/qdox/qdox/1.12.1/qdox-1.12.1.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/jline/jline/2.14.6/jline-2.14.6.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/junit/junit/4.12/junit-4.12.pom]
Failed to get resource: GET. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/junit/platform/junit-platform-launcher/1.3.1/junit-platform-launcher-1.3.1.pom]
Failed to get resource: GET. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/apache/ant/ant-launcher/1.9.13/ant-launcher-1.9.13.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/apache/ant/ant-parent/1.9.13/ant-parent-1.9.13.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/hamcrest/hamcrest-core/1.3/hamcrest-core-1.3.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/apiguardian/apiguardian-api/1.0.0/apiguardian-api-1.0.0.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/junit/platform/junit-platform-engine/1.3.1/junit-platform-engine-1.3.1.pom]
Found locally available resource with matching checksum: [https://repo.maven.apache.org/maven2/org/junit/platform/junit-platform-engine/1.3.1/junit-platform-engine-1.3.1.pom, /Users/marc/.m2/repository/org/junit/platform/junit-platform-engine/1.3.1/junit-platform-engine-1.3.1.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/opentest4j/opentest4j/1.1.1/opentest4j-1.1.1.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/junit/platform/junit-platform-commons/1.3.1/junit-platform-commons-1.3.1.pom]
Found locally available resource with matching checksum: [https://repo.maven.apache.org/maven2/org/junit/platform/junit-platform-commons/1.3.1/junit-platform-commons-1.3.1.pom, /Users/marc/.m2/repository/org/junit/platform/junit-platform-commons/1.3.1/junit-platform-commons-1.3.1.pom]
Found locally available resource with matching checksum: [https://repo.maven.apache.org/maven2/org/opentest4j/opentest4j/1.1.1/opentest4j-1.1.1.pom, /Users/marc/.m2/repository/org/opentest4j/opentest4j/1.1.1/opentest4j-1.1.1.pom]
Resource missing. [HTTP HEAD: https://oss.jfrog.org/artifactory/oss-snapshot-local/org/codehaus/groovy/groovy-all/2.5.3-SNAPSHOT/groovy-all-2.5.3-20180926.165650-42.jar]
Found locally available resource with matching checksum: [https://repo.maven.apache.org/maven2/org/junit/platform/junit-platform-engine/1.3.1/junit-platform-engine-1.3.1.jar, /Users/marc/.m2/repository/org/junit/platform/junit-platform-engine/1.3.1/junit-platform-engine-1.3.1.jar]
Found locally available resource with matching checksum: [https://repo.maven.apache.org/maven2/org/junit/platform/junit-platform-commons/1.3.1/junit-platform-commons-1.3.1.jar, /Users/marc/.m2/repository/org/junit/platform/junit-platform-commons/1.3.1/junit-platform-commons-1.3.1.jar]
Found locally available resource with matching checksum: [https://repo.maven.apache.org/maven2/org/opentest4j/opentest4j/1.1.1/opentest4j-1.1.1.jar, /Users/marc/.m2/repository/org/opentest4j/opentest4j/1.1.1/opentest4j-1.1.1.jar]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/apache/ant/ant/1.9.13/ant-1.9.13.pom]
Failed to get resource: GET. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/apache/ant/ant-junit/1.9.13/ant-junit-1.9.13.pom]
Failed to get resource: GET. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/apache/ant/ant-antlr/1.9.13/ant-antlr-1.9.13.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/apache/ant/ant-launcher/1.9.13/ant-launcher-1.9.13.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/apache/ant/ant-parent/1.9.13/ant-parent-1.9.13.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/apache/ant/ant-parent/1.9.13/ant-parent-1.9.13.pom]
Task ':compileGroovy' is not up-to-date because:
  No history is available.
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/apache/ant/ant/1.9.13/ant-1.9.13.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/apache/ant/ant-antlr/1.9.13/ant-antlr-1.9.13.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/apache/ant/ant-launcher/1.9.13/ant-launcher-1.9.13.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/apache/ant/ant-junit/1.9.13/ant-junit-1.9.13.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/apache/ant/ant-launcher/1.9.13/ant-launcher-1.9.13.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/apache/ant/ant-antlr/1.9.13/ant-antlr-1.9.13.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/apache/ant/ant-junit/1.9.13/ant-junit-1.9.13.pom]
Failed to get resource: HEAD. [HTTP HTTP/1.1 409 : https://oss.jfrog.org/artifactory/oss-snapshot-local/org/apache/ant/ant/1.9.13/ant-1.9.13.pom]
Compiling with JDK Java compiler API.
startup failed:
/Users/marc/Repositories/gradle/gradle/build/tmp/test files/InProcessGroovyCompilerIntegrationTest/jointCompileBadCode...rDisabled/f1nto/src/main/groovy/Groovy.groovy: 1: unable to resolve class Bad 
 @ line 1, column 58.
   class Groovy { void m() { Bad code = new
                                 ^

/Users/marc/Repositories/gradle/gradle/build/tmp/test files/InProcessGroovyCompilerIntegrationTest/jointCompileBadCode...rDisabled/f1nto/src/main/groovy/Groovy.groovy: 1: unable to resolve class thatDoesntAffectStubGeneration 
 @ line 1, column 65.
   roovy { void m() { Bad code = new thatDo
                                 ^

2 errors




> Task :compileGroovy FAILEDFAILURE: Build failed with an exception.

:compileGroovy (Thread[Execution worker for ':' Thread 6,5,main]) completed. Took 18.315 secs.
* What went wrong:
4 actionable tasks: 4 executed
Execution failed for task ':compileGroovy'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --debug option to get more log output. Run with --scan to get full insights.

* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':compileGroovy'.
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:96)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:65)
	at org.gradle.api.internal.tasks.execution.ActionEventFiringTaskExecuter.execute(ActionEventFiringTaskExecuter.java:44)
	at org.gradle.api.internal.tasks.execution.TimeoutTaskExecuter.execute(TimeoutTaskExecuter.java:53)
	at org.gradle.api.internal.tasks.execution.SnapshotAfterExecutionTaskExecuter.execute(SnapshotAfterExecutionTaskExecuter.java:38)
	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:61)
	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:101)
	at org.gradle.api.internal.tasks.execution.FinalizeInputFilePropertiesTaskExecuter.execute(FinalizeInputFilePropertiesTaskExecuter.java:44)
	at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:91)
	at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:62)
	at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:55)
	at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:54)
	at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:34)
	at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.run(EventFiringTaskExecuter.java:51)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:300)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:292)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:174)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:90)
	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
	at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter.execute(EventFiringTaskExecuter.java:46)
	at org.gradle.execution.plan.LocalTaskNodeExecutor.execute(LocalTaskNodeExecutor.java:47)
	at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:303)
	at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:288)
	at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker$1.execute(DefaultPlanExecutor.java:134)
	at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker$1.execute(DefaultPlanExecutor.java:129)
	at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.execute(DefaultPlanExecutor.java:202)
	at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.executeNextNode(DefaultPlanExecutor.java:193)
	at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.run(DefaultPlanExecutor.java:129)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
Caused by: org.gradle.api.internal.tasks.compile.CompilationFailedException: Compilation failed; see the compiler error output for details.
	at org.gradle.api.internal.tasks.compile.ApiGroovyCompiler.execute(ApiGroovyCompiler.java:184)
	at org.gradle.api.internal.tasks.compile.ApiGroovyCompiler.execute(ApiGroovyCompiler.java:57)
	at org.gradle.api.internal.tasks.compile.GroovyCompilerFactory$DaemonSideCompiler.execute(GroovyCompilerFactory.java:77)
	at org.gradle.api.internal.tasks.compile.GroovyCompilerFactory$DaemonSideCompiler.execute(GroovyCompilerFactory.java:65)
	at org.gradle.api.internal.tasks.compile.daemon.AbstractDaemonCompiler$CompilerCallable.call(AbstractDaemonCompiler.java:88)
	at org.gradle.api.internal.tasks.compile.daemon.AbstractDaemonCompiler$CompilerCallable.call(AbstractDaemonCompiler.java:76)
	at org.gradle.workers.internal.DefaultWorkerServer.execute(DefaultWorkerServer.java:42)
	at org.gradle.workers.internal.DefaultWorkerServer.execute(DefaultWorkerServer.java:25)
	at org.gradle.workers.internal.IsolatedClassloaderWorkerFactory$WorkerCallable.call(IsolatedClassloaderWorkerFactory.java:175)
	at org.gradle.workers.internal.IsolatedClassloaderWorkerFactory.executeInWorkerClassLoader(IsolatedClassloaderWorkerFactory.java:104)
	at org.gradle.workers.internal.IsolatedClassloaderWorkerFactory.access$100(IsolatedClassloaderWorkerFactory.java:54)
	at org.gradle.workers.internal.IsolatedClassloaderWorkerFactory$1$1.call(IsolatedClassloaderWorkerFactory.java:78)
	at org.gradle.workers.internal.IsolatedClassloaderWorkerFactory$1$1.call(IsolatedClassloaderWorkerFactory.java:75)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:314)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:304)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:174)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:100)
	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.call(DelegatingBuildOperationExecutor.java:36)
	at org.gradle.workers.internal.IsolatedClassloaderWorkerFactory$1.execute(IsolatedClassloaderWorkerFactory.java:75)
	at org.gradle.workers.internal.IsolatedClassloaderWorkerFactory$1.execute(IsolatedClassloaderWorkerFactory.java:70)
	at org.gradle.api.internal.tasks.compile.daemon.AbstractDaemonCompiler.execute(AbstractDaemonCompiler.java:56)
	at org.gradle.api.internal.tasks.compile.NormalizingGroovyCompiler.delegateAndHandleErrors(NormalizingGroovyCompiler.java:122)
	at org.gradle.api.internal.tasks.compile.NormalizingGroovyCompiler.execute(NormalizingGroovyCompiler.java:54)
	at org.gradle.api.internal.tasks.compile.NormalizingGroovyCompiler.execute(NormalizingGroovyCompiler.java:39)
	at org.gradle.api.internal.tasks.compile.AnnotationProcessorDiscoveringCompiler.execute(AnnotationProcessorDiscoveringCompiler.java:51)
	at org.gradle.api.internal.tasks.compile.AnnotationProcessorDiscoveringCompiler.execute(AnnotationProcessorDiscoveringCompiler.java:37)
	at org.gradle.api.internal.tasks.compile.CleaningJavaCompilerSupport.execute(CleaningJavaCompilerSupport.java:39)
	at org.gradle.api.internal.tasks.compile.CleaningJavaCompilerSupport.execute(CleaningJavaCompilerSupport.java:27)
	at org.gradle.api.tasks.compile.GroovyCompile.compile(GroovyCompile.java:76)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:73)
	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.doExecute(StandardTaskAction.java:48)
	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:41)
	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:28)
	at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:704)
	at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:671)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$1.run(ExecuteActionsTaskExecuter.java:117)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:300)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:292)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:174)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:90)
	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:106)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:85)
	... 33 more


* Get more help at https://help.gradle.org

BUILD FAILED in 25s

Condition not satisfied:

!groovyClassFile('Groovy.class').exists()
||                               |
||                               true
|/Users/marc/Repositories/gradle/gradle/build/tmp/test files/InProcessGroovyCompilerIntegrationTest/jointCompileBadCode...rDisabled/f1nto/build/classes/groovy/main/Groovy.class
false


	at org.gradle.groovy.compile.BasicGroovyCompilerIntegrationSpec.jointCompileBadCodeWithAnnotationProcessorDisabled(BasicGroovyCompilerIntegrationSpec.groovy:299)
```

Reply:
That's a bug in our ApiGroovyCompiler then - it should check for proc:none.

Reply:
I added a check in 0ecba03949d.